x/capability/keeper/keeper.go

```diff
@@ -114,6 +114,12 @@ func (k *Keeper) InitializeAndSeal(ctx sdk.Context) {
 	k.sealed = true
 }
 
+// GetLatestIndex returns the latest index of the CapabilityKeeper
+func (k Keeper) GetLatestIndex(ctx sdk.Context) uint64 {
```

Comment:
Created this function to use for testing purposes. Don't believe there's any issues exposing it. lmk if i'm wrong

Reply:
why not move it to `keeper_test.go` if it's used for testing?

Reply:
`getLatestIndex(k keeper.Keeper, ctx sdk.Context) uint64`

Reply:
It's a function on the Keeper, keeper_test is a different package

Reply:
Don't believe that's possible since that requires access to currently private fields